create-reporter/index.test.js

```diff
@@ -1,3 +1,10 @@
+let mockOS = jest.requireActual('os')
+jest
+  .spyOn(mockOS, 'platform')
```

Comment:
`spyOn` needs to then test that somebody called this method. Since we do not need it here, there is a better alternative:

```js
os.platform = () => 'linux'
```

Reply:
`os`? not `mockOS`?

Reply:
Yeap, let’s call it more clear

Reply:
If I call a variable `os`, I get an error `The module factory of `jest.mock()` is not allowed to reference any out-of-scope variables.`

Reply:
It is a pretty dangerous warning. I am not sure, that `mockOS` should work too.

`jest.mock` is not a function, it is a command for Babel to put its content and move it to the file replacement.

Let’s do the simple way:

```js

jest.mock('os', () => {
  return {
    platform: 'linux'
  }
})
```

Reply:
I tried this before.
I get something like `TypeError: os.hostname is not a function` in other tests

Reply:
```js
let mockOS = jest.requireActual('os')
mockOS.platform = () => 'linux'
jest.mock('os', () => mockOS)
```

this works

Reply:
```js
jest.mock('os', () => {
  return {
    hostname: () => 'localhost'
    platform: () => 'linux'
  }
})
```

The problem with `mockOS` that it should not work :(.